libs/util/code-gen-types/src/dotnet-plugin-events-params.types.ts

```diff
@@ -63,6 +63,12 @@ export interface CreateEntityModelParams extends EventParams {
   apisDir: string;
 }
 
+export interface createResourceDbContextFileParams extends EventParams {
+  entities: Entity[];
+  resourceName: string;
+  apisDir: string;
```

Comment:
Is `apisDir` needed? Or should it be just context folder?

Reply:
@overbit  We are passing it to the internal method, so I think we need to pass it to the plugin event too:

<img width="617" alt="image" src="https://github.com/amplication/amplication/assets/97830649/94fd4c2e-a6f9-4895-a4ea-1c5f45761772">

Reply:
Let's use the same name then. Apisdir is a bit cryptic

Reply:
Done